web-gateway/conf/application.conf

```diff
@@ -1,2 +1,2 @@
 play.crypto.secret = "somesecret"
-
+play.instruction.show = true
```

Comment:
For a future improvement: we could use a different namespace since `play.xyz` is meant for play, while this setting is specific to `online-auction` app.